package.json

```diff
@@ -5,6 +5,8 @@
   "productName": "Git-it",
   "main": "main.js",
   "scripts": {
+    "app": "electron .",
+    "package": "electron-packager . Git-it --platform=win32,darwin --arch=x64 --version=0.30.0 --overwrite --icon=assets/git-it --ignore='node_modules/electron-prebuild'",
```

Comment:
`electron-packager` ignores `electron-prebuilt` by default, fun fact, so we only need `--ignore='node_modules/standard'` because that was the biggest of the app's node modules.